### DISCUSSION TITLE
Save Default Audio Devices

### DISCUSSION THREAD

**Discussion by ibigfire** [File-New-Project/EarTrumpet#1535] [Ideas]
If this is out of scope I totally get it, but since you already have the ability to change audio devices maybe this is possible?  When I install a new audio device, Windows likes to change it to the default.  It would be nice if EarTrumpet could let me instead lock an audio device as the default and not have Windows change it or at least be able to conveniently change it back to my preferred device, both for communication and/or regular.  And recording device as well.

Might tie in to https://github.com/File-New-Project/EarTrumpet/discussions/659 though is slightly different, as being able to just keep it without a hotkey would be ideal.

Obviously it would be nice if Windows just didn't do this in the first place, but I have much higher faith in third party developers actually making a useful customization like this possible.

Either way though, thanks for the consideration and for the very useful app!

**Comment by Hotohori** [comment]
Yeah, same problem here. I bought a new monitor, every damn time I switch it on Windows switch the audio to the monitor what it shouldn't do, so every time I need to switch it manually back to my Realtek device.
